app/scripts/views/sign_up.js

```diff
@@ -90,13 +91,43 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
         });
     },
 
+    _createCoppaInputView: function () {
+      var self = this;
+
+      if (self._coppa) {
+        return p();
+      }
+
+      var coppaDatePicker = new CoppaAgeInput({
+        el: self.$('#coppa'),
+        formPrefill: self._formPrefill,
+        screenName: self.getScreenName()
+      });
+
+      return coppaDatePicker.render()
```

Comment:
Lines 107-113 mirror lines 85-91. Maybe both of these functions should just create the view, and allow `_createCoppa` to call render & the rest.